### ISSUE TITLE
Eliminate import-time db queries

### ISSUE THREAD

**Issue by aaron-kaplan** [iridl/python-maprooms#242]
because they cause the app to fail to start if the db happens to be unavailable at that moment.

**Comment by remicousin** [comment]
Is that also resolved by the "registration" PRs?

**Comment by aaron-kaplan** [comment]
Yes. I tested it by setting db.host to blahblah in config-dev.yaml; the application starts up despite that.
